tensorflow_datasets/scripts/document_dataset_with_example.py

```diff
@@ -0,0 +1,39 @@
+"""
+Script used to generate documentation .md files with example figures
+
+This script uses scripts/generate_visualization.py to generate
+an example figure of a dataset, using show_examples() it gets a figure
+that is temporary stores in examples/ and later used when generating
+the documentation with document_single_builder(builder)
+
+Typical usage example:
+
+  ds_name = 'beans'
+  generate_docs_with_visualization(ds_name)
+
+"""
+
+import os
+import tensorflow_datasets as tfds
+from absl import app
+from tensorflow_datasets.scripts import document_datasets
+
+FIG_DIR = ('../docs/catalog/')
```

Comment:
Images should be saved in a subfolder, like `docs/catalog/images/`

Reply:
Done